core/src/main/kotlin/com/malinskiy/marathon/execution/progress/PoolProgressAccumulator.kt

```diff
@@ -19,11 +19,11 @@ import kotlin.math.roundToInt
 
 class PoolProgressAccumulator(
     private val poolId: DevicePoolId,
-    shard: TestShard,
-    private val configuration: Configuration,
+    val shard: TestShard,
+    val configuration: Configuration,
```

Comment:
why the change to public?

Reply:
fixed

Reply:
Where? I still see `val configuration: Configuration,`

Reply:
Please check the file. You see the outdated information (GitHub specific).